### PR TITLE
chore(release): v0.3.0

### PR DESCRIPTION
**Pull request by theborakompanioni**
Prepares the v0.3.0 release.

### CODE REVIEW

**Review by editwentyone — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+* validate mnemonic phrase against BIP39 wordlist ([#739](https://github.com/joinmarket-webui/jam/issues/739)) ([69e8fa7](https://github.com/joinmarket-webui/jam/commit/69e8fa7f1218fffb83d1c372d39c2264de909b63))
+
+#### Fixed
+
+* allow absolute maker fee of zero ([#727](https://github.com/joinmarket-webui/jam/issues/727)) ([c04a830](https://github.com/joinmarket-webui/jam/commit/c04a8304dbbce6a2dbfdfbada22317a97815dc3e))
+* amount input field to properly interpret BTC ([#800](https://github.com/joinmarket-webui/jam/issues/800)) ([072a419](https://github.com/joinmarket-webui/jam/commit/072a41950d7efab144bab079be3641df8095c985))
+* **earn:** validate offer minsize ([#745](https://github.com/joinmarket-webui/jam/issues/745)) ([7aef192](https://github.com/joinmarket-webui/jam/commit/7aef192e72de95c8cc3b20e0a76f369e98208318))
+* **fb:** months order across timezones ([#853](https://github.com/joinmarket-webui/jam/issues/853)) ([d2116f5](https://github.com/joinmarket-webui/jam/commit/d2116f5ce703f8e7790e55d6179899cfa925fe14))
+* **fb:** display error alert in modal  ([fef6260](https://github.com/joinmarket-webui/jam/commit/fef6260be03d1fe0bf65e1b6fafb8f10dbf8d2af))
```
Comment (MarnixCroes) [inline CHANGELOG.md:27]:
```suggestion
* **fb:** display error alert in modal  ([#777](https://github.com/joinmarket-webui/jam/pull/776)) ([fef6260](https://github.com/joinmarket-webui/jam/commit/fef6260be03d1fe0bf65e1b6fafb8f10dbf8d2af))
```
the PR link is missing here

**Review by MarnixCroes — COMMENTED** [review]
ain't this one missing? https://github.com/joinmarket-webui/jam/pull/807

**Comment by editwentyone** [comment]
i tested it and it was there when I clicked sweep

**Review by editwentyone — APPROVED** [review]

**Comment by MarnixCroes** [comment]
> ain't this one missing? https://github.com/joinmarket-webui/jam/pull/807

Sry maybe I was not clear. I meant that it is missing in the changelog, while it is worth mentioning imo

**Pull request merged**